pr_agent/algo/utils.py

```diff
@@ -192,23 +192,21 @@
             if is_value_no(value):
                 if gfm_supported:
                     markdown_text += f"<tr><td>"
-                    markdown_text += f"{emoji}&nbsp;<strong>No key issues to review</strong>"
+                    markdown_text += f"{emoji}&nbsp;<strong>No major issues detected</strong>"
                     markdown_text += f"</td></tr>\n"
                 else:
-                    markdown_text += f"### {emoji} No key issues to review\n\n"
+                    markdown_text += f"### {emoji} No major issues detected\n\n"
             else:
-                # issues = value.split('\n- ')
-                issues =value
-                # for i, _ in enumerate(issues):
-                #     issues[i] = issues[i].strip().strip('-').strip()
+                issues = value
                 if gfm_supported:
                     markdown_text += f"<tr><td>"
-                    markdown_text += f"{emoji}&nbsp;<strong>{key_nice}</strong><br><br>\n\n"
+                    # markdown_text += f"{emoji}&nbsp;<strong>{key_nice}</strong><br><br>\n\n"
+                    markdown_text += f"{emoji}&nbsp;<strong>Recommended focus areas for review</strong><br><br>\n\n"
                 else:
-                    markdown_text += f"### {emoji} Key issues to review\n\n#### \n"
+                    markdown_text += f"### {emoji} Recommended focus areas for review\n\n#### \n"
                 for i, issue in enumerate(issues):
                     try:
-                        if not issue:
+                        if not issue or not isinstance(issue, dict):
                             continue
```

Comment:
**Suggestion:** Consider adding a comment explaining why the check for 'not isinstance(issue, dict)' is necessary. This will help future developers understand the purpose of this condition and potential edge cases it's handling. [maintainability, importance: 6]
```suggestion
                        # Skip empty issues or non-dictionary items to ensure valid data structure
                        if not issue or not isinstance(issue, dict):
                            continue
```